public/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <script src="https://cdn.auth0.com/js/auth0-spa-js/2.0/auth0-spa-js.production.js"></script>
+    <script src="./js/globalConfigs.js"></script>
+    <script src="./js/fetchViewers.js"></script>
+    <script src="./js/encoder.js"></script>
+    <script src="./js/auth.js"></script>
+    <script src="./js/utils/token.js"></script>
+    <script src="./js/videoClientCore.js"></script>
+    <script src="./js/utils/names.js"></script>
+    <script src="./js/manifestPlayer.js"></script>
+    <style>
+      .highlight {
+        background-color: #007FFF;
+        color: #fff;
+      }
+    </style>
+    <h1>Live Streaming Demo App</h1>
+    <div id="broadcaster">
+      <div id="api-controls">
+        <p>Auth type: <span id="auth-type"></span></p>
+        <p>Project ID: <span id="project-id"></span></p>
+        <button id="create-stream">Create Stream</button>
+        <p id="stream-id-display">Stream ID: <span id="stream-id"></span></p>
+      </div>
+      <div id="auth0-controls">
```

Comment:
We probable should add comments somewhere in this to let them know what each button is doing.